spec/services/quotes_service_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe QuotesService do
+  describe '::search_quotes(query)' do
+    it 'returns a parsed hash from the API' do
+      query = 'I am'
+      result = QuotesService.search_quotes(query)
+      expect(result).to be_a Hash
+      expect(result).to have_key :count
+      expect(result).to have_key :results
+      expect(result[:results]).to be_an Array
```

Comment:
go into at least one of the elements in this results array and test the content, tags, and author keys that your code depends on.

Reply:
Ex: `expect(results[:results][0]).to have_key :content`